core/trino-main/src/test/java/io/trino/execution/TestCreateSchemaTask.java

```diff
@@ -36,6 +36,7 @@
 import java.util.concurrent.CopyOnWriteArrayList;
 
 import static io.airlift.concurrent.MoreFutures.getFutureValue;
+import static io.trino.execution.CreateSchemaTask.internalExecute;
```

Comment:
I would avoid static import here. `internalExecute` name alone is very ambiguous.